src/cls/rgw/cls_rgw.cc

```diff
@@ -2200,7 +2200,7 @@ static int rgw_bi_get_op(cls_method_context_t hctx, bufferlist *in, bufferlist *
       idx = op.key.name;
       break;
     case InstanceIdx:
-      encode_obj_index_key(op.key, &idx);
+      encode_obj_versioned_data_key(op.key, &idx);
```

Comment:
@dongbula Just skipped check for key.instance.empty() with this change.
Are we sure we would not get null instance here?